app/src/main/java/org/thoughtcrime/securesms/groups/compose/Components.kt

```diff
@@ -0,0 +1,169 @@
+
+
+@Composable
+fun GroupMinimumVersionBanner(modifier: Modifier = Modifier) {
+    Box(
+        modifier = modifier
+            .fillMaxWidth()
+            .background(LocalColors.current.warning)
+    ) {
+        Text(
+            text = stringResource(R.string.groupInviteVersion),
+            color = LocalColors.current.textAlert,
+            style = LocalType.current.small,
+            maxLines = 2,
+            textAlign = TextAlign.Center,
+            modifier = Modifier.padding(horizontal = 24.dp, vertical = 4.dp)
+        )
+    }
+}
+
+fun LazyListScope.multiSelectMemberList(
+    contacts: List<ContactItem>,
+    modifier: Modifier = Modifier,
+    onContactItemClicked: (accountId: AccountId) -> Unit,
+    enabled: Boolean = true,
+) {
+    items(contacts.size) { index ->
+        val contact = contacts[index]
+        Column(modifier = modifier) {
+            if (index == 0) {
+                // Show top divider for the first item only
+                HorizontalDivider(color = LocalColors.current.borders)
+            }
+
+            RadioButton(
+                onClick = { onContactItemClicked(contact.accountID) },
+                selected = contact.selected,
+                enabled = enabled,
+                contentPadding = PaddingValues(
+                    vertical = LocalDimensions.current.xxsSpacing,
+                    horizontal = LocalDimensions.current.smallSpacing
+                )
+            ) {
+                ContactPhoto(contact.accountID)
+                Spacer(modifier = Modifier.size(LocalDimensions.current.smallSpacing))
+                MemberName(name = contact.name)
+            }
+
+            HorizontalDivider(color = LocalColors.current.borders)
+        }
+    }
+}
+
+@Composable
+fun RowScope.MemberName(
+    name: String,
+    modifier: Modifier = Modifier
+) = Text(
+    text = name,
+    style = LocalType.current.h8,
+    color = LocalColors.current.text,
+    modifier = modifier
+        .weight(1f)
+        .align(CenterVertically)
+)
+
+
+@Composable
+fun ContactPhoto(sessionId: AccountId) {
+    return if (LocalInspectionMode.current) {
+        Image(
+            painterResource(id = R.drawable.ic_profile_default),
+            colorFilter = ColorFilter.tint(LocalColors.current.textSecondary),
+            contentScale = ContentScale.Inside,
+            contentDescription = null,
+            modifier = Modifier
+                .size(48.dp)
+                .clip(CircleShape)
+                .border(1.dp, LocalColors.current.borders, CircleShape)
+        )
+    } else {
+        val context = LocalContext.current
+        // Ideally we migrate to something that doesn't require recipient, or get contact photo another way
+        val recipient = remember(sessionId) {
+            Recipient.from(context, Address.fromSerialized(sessionId.hexString), false)
+        }
+        Avatar(recipient, modifier = Modifier.size(48.dp))
```

Comment:
The 48.dp here and on line 130 can be moved to the LocalDimensions. Maybe as another icon value (I mentioned this for the 24.dp value somewhere else )